docs/antora.yml

```diff
@@ -0,0 +1,32 @@
+name: migrations 
+version: ~
```

Comment:
`~` means that your documentation component is unversioned. You can define a version at a later point in time, for instance when you will have a maintenance branch for 2.x and a main branch for 3.x.

Reply:
So, I introduced a 1.x branch for the 1.x stuff I am gonna maintain. In that branch it shall stick with the old way of doing things.

I will continue releasing main (it is still named `main`, but its version is 2.x-SNAPSHOT). What do I need to add here @Mogztter ?

Reply:
It depends on how you will manage versions. Antora recommends to create a maintenance branch per minor version.
So, the first version will be "2.0", "2.1", "2.2", "3.0"...

If `main` is 2.x-SNAPSHOT, you can use:

```yaml
version: '2.0-snapshot'
prerelease: true
```